app/assets/stylesheets/responsive.scss

```diff
@@ -25,4 +25,7 @@
   #usernav {
     display: none;
   }
+  #initiative-branding {
+    display: block;
```

Comment:
Can you make this `display: flex`? It's not vertically centered for me as `block`.